codalab/lib/bundle_cli.py

```diff
@@ -1469,14 +1473,6 @@ def copy_bundle(
         else:
             unpack = False
 
-        # Bundles stuck in non-final states such as 'running' should not keep
```

Comment:
I'm a bit confused—what will the behavior of `wadd` across instances for bundles in non-final states?

Reply:
What do we want the behavior to be?

Reply:
@nelson-liu I think currently we don't copy bundles that not in final state, see these lines [L1437-L1443](https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/bundle_cli.py#L1437-L1443).

Reply:
oh oops, sorry for missing that! Yeah, I think that it's fine to just omit bundles that aren't in final states.

Reply:
@nelson-liu no worries! I found the logic here a bit confusing as well when I was playing around. The only thing it changed is that bundle in `killed` state will also be preserved as `killed` instead of `failed`. Not sure how you think about this, but let me know.